modules/extensions-spring-support/src/test/resources/schemas/heisenberg.xsd

```diff
@@ -2202,7 +2202,7 @@
       <xs:extension xmlns:mule="http://www.mulesoft.org/schema/mule/core" base="mule:abstractOperatorType">
         <xs:sequence minOccurs="0" maxOccurs="1"></xs:sequence>
         <xs:attribute type="mule:expressionString" use="optional" default="#[payload]" name="victim"></xs:attribute>
-        <xs:attribute type="mule:expressionString" use="required" name="goodbyeMessage"></xs:attribute>
+        <xs:attribute type="mule:expressionString" use="optional" default="We are done" name="goodbyeMessage"></xs:attribute>
```

Comment:
why?

Reply:
The @Optional annotation was added in order to test the new @Deprecated annotation.